fecgov/openFEC
# upgrade node

NPM 10 was released and it's not compatible with node versions below 18.17. We are currently pulling in the most recent NPM version in the circle config. 


**How to Test**
-pull branch
-nvm install 18.17.1   (should be on npm 9.6.7)
-rm -rf node_modules
-npm i && npm run build
-flask run
